packages/app/app/stacks/page.tsx

```diff
@@ -199,10 +58,14 @@ export default function Page() {
           </Tab.List>
           <Tab.Panels>
             <Tab.Panel>
-              <StacksTable orders={activeOrders(data.orders)} />
+              {stackOrders && (
+                <StacksTable stackOrders={filterActiveOrders(stackOrders)} />
+              )}
             </Tab.Panel>
             <Tab.Panel>
-              <StacksTable orders={completedOrders(data.orders)} />
+              {stackOrders && (
+                <StacksTable stackOrders={filterCompletedOrders(stackOrders)} />
+              )}
```

Comment:
We should show an empty state if there are no orders, what do you think?

Reply:
I think it's fine to not show anything.

<img width="760" alt="image" src="https://github.com/SwaprHQ/stackly-ui/assets/5664434/bd7edc80-c15a-495e-8ae8-e84179561d15">

Looking to this is clear it has no orders yet.

Reply:
I was talking about the page. Since we are hidding the table if there are no orders. Maybe we should have a empty state inside the StacksTable.

Reply:
Yeah, makes sense. I'll talk with Akash to design that.

Will not go into this PR.

Reply:
Added this:

<img width="1237" alt="image" src="https://github.com/SwaprHQ/stackly-ui/assets/5664434/e6955e1c-9d85-4faa-a405-b0327672ffe5">